README.md

```diff
@@ -111,14 +125,15 @@ boost in solvers by using pure BitVector encoding (QF_BV) instead. You can
 use this workaround by pulling the following KLEE branch and recompiling
 Souper:
 
-    $ cd /path/to/souper
-    $ rm -rf third_party/klee
-    $ git clone -b pure-bv-qf https://github.com/rsas/klee third_party/klee
-    $ cd build
-    $ make
+```
+$ cd /path/to/souper
+$ rm -rf third_party/klee
+$ git clone -b pure-bv-qf https://github.com/rsas/klee third_party/klee
```

Comment:
The branch used in `update_deps.sh` is `pure-bv-qf-llvm-5.0-patch`

Reply:
ah, thanks!